Framework/Kernel/inc/MantidKernel/V2D.h

```diff
@@ -76,6 +76,8 @@ class MANTID_KERNEL_DLL V2D {
   V2D operator*(const double factor) const;
   /// Scale this
   V2D &operator*=(const double factor);
+  /// Negate
+  V2D operator-() const noexcept;
```

Comment:
Can this safely be `noexcept`? For instance, is it OK to assume that `-NaN` or `-inf` won't throw anything?

Reply:
Yes. I've added unit tests to make sure the behavior won't change unnoticed.

Reply:
Cool - once the build servers pass this looks good to go from my point of view then